config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: :show
+
+  resources :dish_ingredients, only: [:create, :destroy]
+
+  resources :chefs, only: :show do
+    resources :ingredients, only: :index
```

Comment:
As mentioned in the controller, this is where you could choose to add `controller: "chefs/ingredients"`  if you wanted to use a nested controller for the ingredients page instead of the regular `ingredients#index`